worker-pools.yml

```diff
@@ -2958,6 +2958,40 @@ pools:
               type: SCRATCH
               interface: NVME
           machine_type: c2-standard-4
+  - pool_id: '{pool-group}/t-linux-2404-d2g'
```

Comment:
Another thought...

Most consumers are not going to know what `d2g` is. Should we make the suffix `-docker` instead? I think this might be slightly more confusing for us, but much less confusing for everyone else.

I'm on the fence with this one, but think I'm leaning towards `-docker`.

Reply:
Makes sense, thanks.

Reply:
Yeah that would make more sense. Can pools that enable d2g still run tasks in the worker without using docker? Ah, I guess we only care to enable this on pools we are going to be running docker tasks on.